.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  push:
+    branches: ['master']
+  pull_request:
+    branches: ['master']
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [20.x]
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: set up node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: install dependencies
+        uses: borales/actions-yarn@v4.2.0
+        with:
+          cmd: install
+
+      - name: run tests
+        uses: borales/actions-yarn@v4.2.0
+        with:
+          cmd: test
+          dir: apps/mobile
```

Comment:
In the future we might want to run this at the root level

Reply:
Yes. After we set up yarn to run this at the root level, we will just have to delete the `dir` field